scripts/bundleSize/serviceList.js

```diff
@@ -0,0 +1 @@
+export default ["afaanoromoo", "afrique", "amharic", "arabic", "archive", "azeri", "bengali", "burmese", "cymrufyw", "gahuza", "gujarati", "hausa", "hindi", "igbo", "indonesia", "japanese", "korean", "kyrgyz", "marathi", "mundo", "naidheachdan", "nepali", "news", "newsround", "pashto", "persian", "pidgin", "portuguese", "punjabi", "russian", "scotland", "serbian", "sinhala", "somali", "sport", "swahili", "tamil", "telugu", "thai", "tigrinya", "turkce", "ukchina", "ukrainian", "urdu", "uzbek", "vietnamese", "yoruba", "zhongwen"];
```

Comment:
Is there any way we can avoid doing this, as it would be good to have less places to have to configure and maintain services. 

We could use the https://github.com/bbc/simorgh/blob/9997384bcad94b0175a7b8123b373788492f0183/src/app/lib/config/services/loadableConfig.ts

if not, would it be possible to somehow use the Services global type since it's the definitive list of them all? https://github.com/bbc/simorgh/blob/705da8ea446b6f8e83ef750647bedbca40633067/src/app/models/types/global.ts#L106

Reply:
So we did try your suggestion of using `loadableConfig.ts` and it did indeed work from the context of tests; the trouble came when we tried to run the bundlesize script on the command line. With us depending on the `loadableConfig.ts` typescript module we then need to typescript compile the script into JS to run on the command line. We did get this to work except we had issues resoling `loadableConfig.ts` from outside the `scripts/bundleSize` directory. We felt it wasn't worth the hassle of getting this to work as:
- The work was ultimately meant to be a time limited spike
- We virtually never change our services so updating this list will be extremly infrequent

Would you like us to raise a tech ticket to revisit this at a later point? Just put together a little draft PR with our work in progress: https://github.com/bbc/simorgh/pull/10970